src/modelfree/score_agent.py

```diff
@@ -119,5 +182,6 @@ def main():
     score_ex.run_commandline()
 
 
+
 if __name__ == '__main__':
     main()
```

Comment:
Consider adding to `main()` a logging statement that outputs the Sacred directory for the `FileStorageObserver`? I'm imagining a common use case being running it and then wondering which of the ~100 Sacred directories you should be looking at...

Reply:
I like this idea; added as a log statement